sdk/src/types/block/output/feature/block_issuer.rs

```diff
@@ -73,6 +85,18 @@ impl Ed25519BlockIssuerKey {
     pub fn try_from_bytes(bytes: [u8; Self::LENGTH]) -> Result<Self, Error> {
         Ok(Self(ed25519::PublicKey::try_from_bytes(bytes)?))
     }
+
+    /// Creates a dummy [`Ed25519BlockIssuerKey`] used to calculate a storage score for implicit account addresses.
```

Comment:
Wouldn't probably use the term `dummy` anymore. I forgot to remove it.

Reply:
Agree

Reply:
It's pub(crate) anyway, let's just remove the doc